### DISCUSSION TITLE
Sprite Style

### DISCUSSION THREAD

**Discussion by KrabbyCurse** [Techokami/SonicWorldsNext#57] [Ideas]
All Characters should be Sprite Style

Classic Sprite Style:
Sonic: https://www.spriters-resource.com/custom_edited/sonicthehedgehogcustoms/sheet/143567/
Tails: https://www.spriters-resource.com/genesis_32x_scd/sonicth3/sheet/55419/
Knuckles: https://www.spriters-resource.com/genesis_32x_scd/sonicth3/sheet/54475/
Amy: https://twitter.com/Axanery/status/1672017815811792897?s=20
 

Mod Gen Sprite Style:
Sonic: https://www.spriters-resource.com/custom_edited/sonicthehedgehogcustoms/sheet/111918/
Tails: https://www.deviantart.com/oth305/art/Mod-Gen-Classic-Tails-V4-807232235
Knuckles: https://www.spriters-resource.com/custom_edited/sonicthehedgehogcustoms/sheet/143202/
Amy: https://www.deviantart.com/oth305/art/Mod-Gen-Classic-Amy-Sprite-Sheet-V1-902429689

**Comment by Techokami** [comment]
Uhm, if you want to use a different set of sprites in your game, you're free to edit the existing sprite sheets yourself. We won't be providing any alternate sheets.
